sklearn/neighbors/base.py

```diff
@@ -646,6 +645,29 @@ class from an array representing our data set and ask who's
                 return dist, neigh_ind
             return neigh_ind
 
+    def reduce_func(self, radius, return_distance, dist):
```

Comment:
There is a weird problem with this, that this works when I name it as something other than `_reduce_func`. When using `_reduce_func`, it somehow refers to the `_reduce_func` of `KNeighborsMixin` as when I pass `n_neighbors` to it, it accepts.